.goreleaser.yml

```diff
@@ -14,6 +14,29 @@ builds:
       - CGO_ENABLED=0
     binary: falco-exporter
 
+dockers:
+  -
+    goos: linux
+    goarch: amd64
+    skip_push: false
+    dockerfile: build/Dockerfile
+    image_templates:
+      - "falcosecurity/falco-exporter:latest"
+      - "falcosecurity/falco-exporter:{{ .Version }}"
+      - "falcosecurity/falco-exporter:{{ .Major }}.{{ .Minor }}"
```

Comment:
So, supposing tag is `v0.2.0`, this will publish `falco-exporter:v0.2.0` and `falco-exporter:0.2` too?

It this what we want?

Reply:
I would like to publish - assuming tag is `v0.2.0`,  the following image tags:
- `0.2.0`
- `0.2`
- `latest`

Note that `{{ .Version }}` does not include the leading `v`, `goreleaser` already strip it.  So it works already.
Anyway I still have to remove the `snapshot` configuration and enable publish. It's still wip.

Reply:
I do not agree on publishing the minor one for clarity.

Otherwise, we'll have to explain users that an hypothetical 0.2.3 version gets released into 0.2 docker image too since it's a hotfix.

Why not stick with simple things?

Everyone know what `latest` means.
Matching github release to one docker image tag is simpler to understand and communicate.

Just that :)

Reply:
You convinced me. Also because we can add this later. I'm gonna remove it and the snapshot configuration too.